include/uapi/sound/sof/abi.h

```diff
@@ -26,7 +26,7 @@
 
 /* SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
-#define SOF_ABI_MINOR 5
```

Comment:
@bardliao I am fine with the changes but this would need to be updated once we get the kernel updates for MINOR6.
@wenqingfu all kernel PRs that touch the ABI will have to be serialized, and are now gated by the MINOR6 issues.